cosmos/dbt/parser/project.py

```diff
@@ -216,6 +249,10 @@ def __post_init__(self) -> None:
         for file_name in self.models_dir.rglob("*.sql"):
             self._handle_sql_file(file_name)
 
+        # crawl the models in the project
+        for file_name in self.models_dir.rglob("*.py"):
+            self._handle_sql_file(file_name)
```

Comment:
wonder if this should be a separate `_handle_python_file`?

Reply:
@jlaneve we could. I was apprehensive about the low code coverage of this part of the code and our short timelines. Would it be okay if I log a ticket and we refactor this before 1.1? This would give us more time to improve the test coverage of this part of the code.

Reply:
yes, that's good with me!

Reply:
Thanks, @jlaneve , I just added the follow up ticket: #384 384